<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTractionPage</class>
 <widget class="QWidget" name="SetTractionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color: rgb(51,153,255);
border:0px;
font:16px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
border-image: url(:/image/buttonUP.jpg);
font:21px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;
}</string>
  </property>
  <widget class="QPushButton" name="btnEle1Seg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电机1
隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEle2Seg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电机2
隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEle4Seg">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电机4
隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEle3Seg">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电机3
隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrk1Cut">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>120</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电制动
1切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTract1Reset">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>210</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>牵引1
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrk2Cut">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>120</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>电制动
2切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTract2Reset">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>210</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>牵引2
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLowPower">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>140</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>低功率测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMend">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>140</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>维护模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelT11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已隔离</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelT22">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已隔离</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelT12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已隔离</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelT21">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已隔离</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
